app/lib/buildKeySignature.ts

```diff
@@ -0,0 +1,21 @@
+import VexFlow, { IRenderContext } from "vexflow";
+const VF = VexFlow.Flow;
```

Comment:
This const should go below any import statements. Again, you can also just use the key-command to order the imports.